tests/integration/targets/backup_tag/tasks/main.yml

```diff
@@ -8,20 +8,17 @@
   block:
 
     - name: Create an AWS Backup Vault so we have something to tag
-      command: 'aws backup create-backup-vault --backup-vault-name {{ backup_vault_name }}'
+      amazon.aws.backup_vault:
+        backup_vault_name: "{{ backup_vault_name }}"
```

Comment:
We shouldn't need anymore the environment block here.

Reply:
Without these variables the task fails with

"msg": "Unable to get vault facts: Unable to locate credentials",

"The amazon.aws.backup_vault module requires a region and none was found in configuration, environment variables or module parameters",

Am I missing anything in my env?

Reply:
> Am I missing anything in my env?

What's missing is adding the modules to the `aws` group: https://github.com/ansible-collections/amazon.aws/blob/main/meta/runtime.yml these environment variables are used as fallbacks but shouldn't be relied on.